### PR TITLE
Introduce DTOs for the imaging context

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

**Review by jenniferboedker — DISMISSED** [review]
LGTM.
@Steffengreiner already checked if all DTOs are present and if the descriptions is correct with the last PR.

**Comment by KochTobi** [comment]
I just noticed that you provide getter and setter methods for some but for others you do not although the fields are `private`. Maybe we should be consistent here and add getter methods and constructors? For example for `AquisitionParameter`.

**Review by KochTobi — CHANGES_REQUESTED** [review]
Some classes do not provide access to set or get their properties.

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/EmissionRange.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * @author: Jennifer Bödker
+ *
+ */
+class EmissionRange {
+
+    /**
+     * Its the smallest wavelength within the emission range
+     * e.g 495 nm
+     */
+    private final int startWaveLength
```
Comment (KochTobi) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/EmissionRange.groovy:18]:
```suggestion
    private final int startWavelength
```
wavelength is one word

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/EmissionRange.groovy [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    /**
+     * Its the biggest wavelength within the emission range
+     * e.g 530 nm
+     */
+    private final int endWaveLength
```
Comment (KochTobi) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/EmissionRange.groovy:24]:
```suggestion
    private final int endWavelength
```

**Review by KochTobi — COMMENTED** [review]
field rename

**Review by KochTobi — APPROVED** [review]

File: src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/EmissionRange.groovy [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    private final int endWavelength
+
+    /**
+     * Class storing the emissions from a light source as a range of wave lengths
+     * @param startWaveLength {@link EmissionRange#startWaveLength}
+     * @param endWaveLength {@link EmissionRange#endWaveLength}
+     */
+    EmissionRange(int startWaveLength, int endWaveLength) {
+        this.startWaveLength = startWaveLength
+        this.endWaveLength = endWaveLength
```
Comment (KochTobi) [inline src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/EmissionRange.groovy:33]:
```suggestion
     * @param startWavelength {@link EmissionRange#startWavelength}
     * @param endWavelength {@link EmissionRange#endWavelength}
     */
    EmissionRange(int startWavelength, int endWavelength) {
        this.startWavelength = startWavelength
        this.endWavelength = endWavelength
```

**Review by KochTobi — CHANGES_REQUESTED** [review]
`wavelength` refactor

**Review by KochTobi — APPROVED** [review]

**Pull request merged**